### PR TITLE
Kevin/Deriv blog - Side CTA Banner

### PR DESCRIPTION
**Pull request by kevinw-binary**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/blog/side-cta/_style.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    padding-left: 16px;
+    background-color: var(--color-grey-41);
+    flex-flow:column;
+    justify-content:flex-start;
+    background-image: url(${Shape});
+    background-position: right bottom; 
+    background-repeat: no-repeat;
+
+    @media ${device.tablet} {
+        width: 328px;
```
Comment (sean-binary) [inline src/pages/blog/side-cta/_style.js:29]:
```suggestion
        width: 100%;
        max-width: 328px;
```

Comment (sean-binary) [reply src/pages/blog/side-cta/_style.js:29]: You probably wanna do something like this instead to cater all the way down to 320px.

File: src/pages/blog/side-cta/_style.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        height: 420px;
+        background-position: center bottom; 
+        padding-top: 26px;
+        padding-left:39px;
+        padding-right:46px;
```
Comment (sean-binary) [inline src/pages/blog/side-cta/_style.js:34]:
```suggestion
        padding: 26px 46px 0 39px;
```

File: src/pages/blog/side-cta/_style.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    @media ${device.tablet} {
+        justify-content:center;
+    }
+`
+export const CtaContainer = styled(Flex)`
+    width: 282px;
+    height: 420px;
+    border-radius: 8px;
+    padding-top: 25px;
+    padding-left: 16px;
```
Comment (sean-binary) [inline src/pages/blog/side-cta/_style.js:20]:
```suggestion
    padding: 25px 0 0 16px;
```

File: src/pages/blog/side-cta/_style.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+import { LinkButton } from 'components/form'
+
+export const SideContainer = styled(Flex)`
+    padding: 20px;
+    justify-content:start;
+
+    @media ${device.tablet} {
+        justify-content:center;
+    }
+`
```
Comment (sean-binary) [inline src/pages/blog/side-cta/_style.js:14]: We can pass all this in as props. For the media query, we can pass in an object with styles to `tablet={{ justifyContent: 'center' }}`

File: src/pages/blog/side-cta/_style.js [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+    background-image: url(${Shape});
+    background-position: right bottom;
+    background-repeat: no-repeat;
+
+    @media ${device.tablet} {
+        width: 100%;
+        max-width: 328px;
+        height: 420px;
+        background-position: center bottom;
+        padding: 26px 46px 0 39px;
```
Comment (njazuli-deriv) [inline src/pages/blog/side-cta/_style.js:32]:
```suggestion
        padding: 24px 39px 0;
```

File: src/pages/blog/side-cta/_style.js [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+    @media ${device.tablet} {
+        justify-content: center;
+    }
+`
+export const CtaContainer = styled(Flex)`
+    width: 282px;
+    height: 420px;
+    border-radius: 8px;
+    padding: 25px 0 0 16px;
```
Comment (njazuli-deriv) [inline src/pages/blog/side-cta/_style.js:19]:
```suggestion
    padding: 24px 0 0 16px;
```

File: src/pages/blog/side-cta/index.js [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+                    <DownloadPdfIcon src={DownloadPdf} />
+                    <Header as="h3" type="subtitle-2" mb="5px">
+                        {pdf_data.title}
+                    </Header>
+                    <Header as="p" type="paragraph-2">
+                        by {pdf_data.author}
+                    </Header>
+
+                    <LinkButtonWrapper>
+                        <ClaimButton tertiary to="/blog/articles/">
```
Comment (sean-binary) [inline src/pages/blog/side-cta/index.js:31]:
```suggestion
                        <ClaimButton tertiary to="/landing/ebooks/stocks/">
```

Comment (sean-binary) [reply src/pages/blog/side-cta/index.js:31]: Let's use this for now.

File: src/pages/blog/side-cta/_style.js (hunk 6 above)
Comment (njazuli-deriv) [reply src/pages/blog/side-cta/_style.js:19]: as per design

File: src/pages/blog/side-cta/_style.js (hunk 5 above)
Comment (njazuli-deriv) [reply src/pages/blog/side-cta/_style.js:32]: as per design

**Review by sean-binary — APPROVED** [review]

**Pull request merged**